src/vs/workbench/contrib/debug/browser/debugToolBar.ts

```diff
@@ -188,10 +193,18 @@ export class DebugToolBar extends Themable implements IWorkbenchContribution {
 		this._register(this.layoutService.onDidChangePartVisibility(() => this.setCoordinates()));
 
 		const resizeListener = this._register(new MutableDisposable());
+		const nextFrameScheduler = this._register(new RunOnceScheduler(() => {
+			const thisWindow = dom.getWindow(this.layoutService.activeContainer);
+			this.setCoordinates(this.getCurrentXPercent() * thisWindow.innerWidth);
+		}, 30));
```

Comment:
Next frame would be 0, why 30?

Reply:
I actually just wanna make some performance tweaks so this isn't necessary.